packages/common/src/hooks/chats/useChatBlastAudienceContent.ts

```diff
@@ -37,12 +37,9 @@ export const useChatBlastAudienceContent = ({ chat }: { chat: ChatBlast }) => {
     },
     { disabled: !decodedContentId || audienceContentType !== 'track' }
   )
-  const { data: album } = useGetPlaylistById(
-    {
-      playlistId: decodedContentId!
-    },
-    { disabled: !decodedContentId || audienceContentType !== 'album' }
-  )
+  const { data: album } = useCollection(decodedContentId!, {
+    enabled: !(!decodedContentId || audienceContentType !== 'album')
```

Comment:
nit: distribute the `!`

Reply:
plus we dont even need to have decodedContentId here since that is handled internally now